site/docs/components/pagination/examples.mdx

```diff
@@ -0,0 +1,96 @@
+
+#### Best practices
+
+- Include the input field when pagination is likely to total hundreds or thousands of pages.
+- Order results logically, such as in a chronological order. This helps users to navigate through several pages at a time.
+- Place the input to the left of the pagination range if users will use it regularly. Otherwise, position it to the right.
+
+</LivePreview>
+
+<LivePreview componentName="pagination" exampleName="Compact" >
+
+### Compact
+
+Use `CompactPaginator` for small viewports, or to minimize visual clutter in a high-density interface. This component only displays the active and last pages. which means the width is more predictable. This provides more flexibility in a layout.
+
+#### Best practices
+
+- Consider this variant when space is limited.
+- Position the compact component appropriately for your UI. Use left aligned for pages of text results, center for pages of images or equally-sized tiles, and right aligned for a full-width data grid.
+
+</LivePreview>
+
+<LivePreview componentName="pagination" exampleName="CompactWithInput" >
+
+### Compact with input
+
+Use `CompactPaginator` with the `withInput` prop to provide the ability to jump to a specific page number.
+
+</LivePreview>
+
+<LivePreview componentName="pagination" exampleName="Truncation" >
+
+### Truncation
+
+The component will truncate the page numbers displayed when it cannot display all of them in a given space. An ellipsis communicates this to the user.
+
+#### Best practices
+
+- Consider the number of results likely to be displayed and the available space, when configuring the width of your Pagination. Although truncation helps minimize its width, a range containing 100s or 1000s of pages will inevitably be wider than a range containing two-digit pages.
+- Allow space for the width to change, when setting horizontal alignment, as it may impact other surrounding elements.
+
+</LivePreview>
+
+<LivePreview componentName="pagination" exampleName="SiblingPairCount" >
+
+### Sibling pair count
+
+You can configure the visible page range using the `siblingPairCount` prop—increasing or decreasing the sibling range, i.e., pages on either side of the active page, depending on the breakpoint. Always make changes to sibling pages in sets of two to ensure that your page range is an odd number. This keeps the active page centred within the range and visually balanced.
+
+</LivePreview>
+
+<LivePreview componentName="pagination" exampleName="BoundaryCount" >
+
+### Boundary count
+
+You can configure the visible page range using the `boundaryCount` prop—increasing or decreasing the number of pages displayed next to the arrow buttons. The minimum value is 1, so there will always be at least one item next to the arrow buttons.
+
+</LivePreview>
+
+<LivePreview componentName="pagination" exampleName="CompactWithGoTo" >
+
+### Compact with GoToInput
+
+This is an example of a custom combination of `CompactPaginator` and `GoToInput`.
```

Comment:
This isn't really guidance, and I'm not sure what a custom combination means in this context.

Reply:
Was an example asked for by Ben. Was based on conversations around Compact with input and wanted to have this one too, with the GoTo so people can compare and make better decisions.

Reply:
how does this sound?